components/Company/Logo.js

```diff
@@ -0,0 +1,9 @@
+import '../../styles/Logo.css'
```

Comment:
![image](https://user-images.githubusercontent.com/27492820/67632221-6e126d00-f8d3-11e9-8462-18ec9efa26ff.png)
dont forget mobile first, responsiveness.

Reply:
yes I have, but I deactivate for the moment. I have set it in next.config.js file. I will use it from now. For the mobile first and exactly for landing page, I haven't done much because I got really confused about UI design, so until now I am still focused on building the rest api. But since Mr. Ihsan gave the brand guidline in trello, I got some inspirations.